src/Java.Interop/Java.Interop-MonoAndroid.csproj

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <XAConfigPath>..\..\bin\Build$(Configuration)\XAConfig.props</XAConfigPath>
```

Comment:
This is a legacy project.  I believe that this `<PropertyGroup/>` should go *after* old line 3/new line 7, as old 4/new 8 *overrides* `$(Configuration)` if it isn't already set.

This would in turn require *moving* `$(DefineConstants)` to be in a separate property group block, so that it can rely on the results of importing `$(XAConfigPath)`.

Reply:
Indeed, I will change it. I must have been testing it with Debug only I think.

The project is still used in https://github.com/xamarin/xamarin-android/blob/a7413a2389886082c3d3422c50a7e6cc84f43d8f/src/Mono.Android/Mono.Android.targets#L25